libs/zephyr-agent/src/tests/ze-agent.spec.ts

```diff
@@ -117,8 +118,11 @@ runner('ZeAgent', () => {
   });
 
   it(
-    'should be deployed to Zephyr',
+    'should deploy Webpack',
     async () => {
+      const app = 'sample-webpack-application';
+      const uuid = `${app}.${appProject}.${appOrg}`;
```

Comment:
each build should save information about it
there are a code which works for CI only - and should fork for any build
basically we should be able to get latest build snapshot by path to a folder with app